docs/tutorials/on-device-training/ios-app.md

```diff
@@ -0,0 +1,987 @@
+    )
+
+    ```
+    That's all! The training artifacts have been generated in the `artifacts` directory. The artifacts are ready to be deployed to the iOS device for training.
+
+
+## Building the iOS application
+### Xcode Setup
+Open Xcode, and create a new project. Select `iOS` as the platform and `App` as the template. Click Next.
+
+![Xcode Setup New Project](../../../images/iOS_speaker_identification_screenshot_1.png)
+
+Enter the project name. Here, we will name the project 'MyVoice', but you can name it anything you prefer. Make sure to select `SwiftUI` as the interface, and `Swift` as the language. Then, click Next.
+
+![Xcode Setup Project Name](../../../images/iOS_speaker_identification_screenshot_2.png)
+
+Select the location where you want to save the project and click `Create`.
+
+
+Now, we will need to add the `onnxruntime-training-objc` pods to the project. We will use `CocoaPods` to add the dependency. If you don't have CocoaPods installed, you can install it by running the following command in the terminal:
+
+```bash
+sudo gem install cocoapods
+```
+
+Navigate to the project directory and run the following command to create a `Podfile`:
+
+```bash
+pod init
+```
+
+This will create a `Podfile` in the project directory. Open the `Podfile` and add the following line after the `use_frameworks!` line:
+
+```ruby
+pod `onnxruntime-training-objc`, `~> 1.16.0`
+```
+
+Save the `Podfile` and run the following command to install the dependencies:
+
+```bash
+pod install
+```
+
+This will create a `MyVoice.xcworkspace` file in the project directory. Open the `xcworkspace` file in Xcode. This will open the project in Xcode with the dependencies installed.
+
+Now, create a new group in the project and name it `artifacts`. Drag and drop the artifacts generated in the previous section into the `artifacts` group. Make sure to select `Create folder references` and `Copy items if needed` options. This will add the artifacts to the project.
+
+Next, create a new group in the project and name it `recordings`. This group will contain the audio recordings that will be used for training. You can generate the recordings by running the [`recording_gen.py`](https://github.com/microsoft/onnxruntime-training-examples/blob/master/on_device_training/mobile/ios/recording_gen.py) script at the root of the project. Alternatively, you can also any other recordings of length 10 seconds in .wav format.
+The project structure should look like this:
+
+![Xcode Project Structure](../../../images/iOS_speaker_identification_screenshot_3.png)
+
+
+### Application Overview
+
+The application will consist of two main UI Views: `TrainingView` and `InferenceView`. The `TrainingView` is used to train the model on-device, and the `InferenceView` is used to perform inference with the trained model. Additionally, there is `ContentView` which is the home view of the application and contains buttons to navigate to the `TrainingView` and `InferenceView`.
```

Comment:
nit: `TrainView` and `InferView` are the names in the code. can just do a find and replace of `TrainingView` and `InferenceView`.